go/chat/utils/utils.go

```diff
@@ -2383,6 +2383,23 @@ func GetUnverifiedConv(ctx context.Context, g *globals.Context, uid gregor1.UID,
 	return inbox.ConvsUnverified[0], nil
 }
 
+func FormatConversationName(info chat1.ConversationInfoLocal, myUsername string) string {
+	if info.TopicName != "" {
+		return fmt.Sprintf("%s#%s", info.TlfName, info.TopicName)
+	}
+	users := strings.Split(info.TlfName, ",")
+	if len(users) > 1 {
+		usersWithoutYou := make([]string, 0, len(users)-1)
```

Comment:
```suggestion
		usersWithoutYou := []string{}
```

Reply:
? That wouldn't prealloc the array (not that it matters for performance of course)

Reply:
yeah it's just more readable.